### ISSUE TITLE
Add keyboard shortcut to focus sidebar

### ISSUE THREAD

**Issue by Vijaynrt851** [a85/POSTMan-Chrome-Extension#6294]
I'm using Windows machine for postman tool

I'm trying to focus on sidebar using keyboard shortcut Ctrl+Alt+1 which is mentioned in postman release notes I have tried with the different combinations of keyboard shortcut But I could not focus on the sidebar every time after I hit the request I need to navigate the request in the collection so I thought of using focus on sidebar shortcut but I am failing. Please let me know whether the shortcut is implemented or not if not when the feature will be implemented

**Comment by numaanashraf** [comment]
@Vijaynrt851 This shortcut is not implemented in the latest version of Postman. We'll consider this as a feature request to add this shortcut back and share any updates here.

**Comment by cWillow** [comment]
Any news on this? It is still documented as a keyboard shortcut

**Comment by xak2000** [comment]
It would be very useful feature!

I have many collections with subfolders (sometimes several levels of depth) that contains multiple requests.

Every time I run Postman, all collections with subfolders are collapsed. But usually I have a tab or two with last executed requests from last used collections. With such "Focus button" or hotkey I will be able to click it or press it while tab with desired request is opened and it will automatically expand given collection and focus this request in the collection. After this it is easy to navigate to other requests in the same collection.

Without this functionality I must find the collection in the collections list, expand it several times, trying to find required subfolder and then find current request (that is already opened in the tab) in some subfolder of the collection. Very inconvenient.

Basically this feature is implemented in some way in all popular code editors:

* Button in the IDEA (have the shortcut too):

  ![image](https://user-images.githubusercontent.com/1576014/91856846-c478d680-ec6f-11ea-9e8b-7121550c60ea.png)

* Button in the Eclipse:

  ![image](https://user-images.githubusercontent.com/1576014/91856876-cd69a800-ec6f-11ea-85ff-358d11fcda59.png)

* `File: Reveal Active File in Sidebar` action in VS Code.

**Comment by johnnyoshika** [comment]
Any update on this? Keyboard shortcuts to focus on sidebar and builder would be really convenient.

**Comment by 7nolikov** [comment]
It was implemented years ago - https://blog.postman.com/speeding-up-workflows-with-keyboard-shortcuts/
Please return this feature, and also please use more convenient combination for this frequently repeated operation

**Comment by bmariusz** [comment]
Any news on that?
